samples/zigbee/shell/README.rst

```diff
@@ -0,0 +1,166 @@
+.. _zigbee_shell_sample:
+
+Zigbee: Shell
+#############
+
+.. contents::
+   :local:
+   :depth: 2
+
+This :ref:`Zigbee <ug_zigbee>` shell sample demonstrates a Zigbee router (with the possibility of being a coordinator) that uses the shell interface inside Zephyr's :ref:`zephyr:shell_api` for interaction.
+
+The sample can be used for several purposes, including:
+
+* Initial configuration of the network - forming a network as a coordinator, adding devices to the network with the install codes, setting the extended PAN ID.
+* Benchmarking - measuring time needed for message to travel from one node to another.
+
+Available CLI interfaces
+************************
+
+The CLI can be accessed by using the following interfaces:
+
+* UART
+* RTT
+
+These interfaces are completely independent one from another and can be used simultaneously or exclusively.
+
+The sample demonstrates the usage of commands listed on :ref:`zigbee_cli_reference`.
+Zigbee CLI is integrated into the system shell that is accessible over the available interfaces.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52840dk_nrf52840, nrf52833dk_nrf52833, nrf5340dk_nrf5340_cpuapp, nrf21540dk_nrf52840
+
+You can use two or more of the development kits listed above and mix different development kits.
+
+Additionally, you can use this sample with any other :ref:`Zigbee <ug_zigbee>` sample application for testing the communication or other sample-specific functionalities.
```

Comment:
Let's move Requirements before the Available CLI interfaces, and Available CLI interfaces under Overview?

Reply:
I do not have a strong opinion on that. Moved.